src/plugins/workspace/server/routes/index.ts

```diff
@@ -46,13 +46,21 @@ const workspaceOptionalAttributesSchema = {
   reserved: schema.maybe(schema.boolean()),
 };
 
+const workspaceNameSchema = schema.string({
+  validate(value) {
+    if (!value || value.trim().length === 0) {
+      return "can't be empty or blank.";
```

Comment:
Not sure if the description is clear enough for customers to understand what happens here. And should we wrap it with i18n?

Reply:
field name will prepend to actual message, the whole error message will be
```json
{
 "statusCode": 400,
 "error": "Bad Request",
 "message": "[request body.attributes.name]: can't be empty or blank."
}
```

>And should we wrap it with i18n?

as the default validation message of [schema.string()](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/packages/osd-config-schema/src/types/string_type.ts#L59)haven't been wrapped by i18n. i think we can keep it same